src/components/HeroImage/HeroImage.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { s } from "./";
+
+export default function HeroImage(props){
+
+	const {
+		idString,
```

Comment:
```suggestion
		id,
```
As mentioned in other comment - `idString` is a bit superflous - kinda like saying `listArray` or `countNo`